docs/docs/guides/security/42crunch.md

```diff
@@ -0,0 +1,81 @@
+# `42Crunch`
```

Comment:
@kylehodgetts @mbana @aabedraba I added a short quite with what needs to be run to have 42Crunch.

Can you please proofread it and tell me where in the menu items i should add this.

Reply:
I will leave it to @aabedraba to add stuff to it if he thinks it is necessary

Reply:
Could you add a short description of what 42Crunch is and why you would want to use it?

Reply:
Done